include/can/core/message_writer.hpp

```diff
@@ -34,8 +34,8 @@ class MessageWriter {
 
   private:
     Writer& writer;
-    std::array<uint8_t, 64> buffer;
```

Comment:
This might be a merge error. I believe I changed 64 to a constant earlier today.

Reply:
I was wrong.